packages/react-select/src/Select.js

```diff
@@ -808,17 +898,12 @@ export default class Select extends Component<Props, State> {
   getElementId = (element: 'group' | 'input' | 'listbox' | 'option') => {
     return `${this.instancePrefix}-${element}`;
   };
-  getActiveDescendentId = () => {
-    const { menuIsOpen } = this.props;
-    const { menuOptions, focusedOption } = this.state;
```

Comment:
I removed this method because it was not being used any more (it's last usage was removed [here](https://github.com/JedWatson/react-select/pull/2581/files#diff-7064e1eb0f7889ae735e705d019ebfa160fbb668d7733f73ee6f107b8cf16fadL1128)) and it wasn't trivial to figure out how it should work with some of the changes I made.